kuka_kr16_support/urdf/kr16_2_macro.xacro

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:property name="DEG2RAD" value="0.017453292519943295"/>
```

Comment:
We need to be careful these kind of properties don't clash with others that may be defined outside this xacro (when used in a composite xacro).

We can't move the property def into the `xacro:macro` itself, so perhaps we should namespace the property (at least for now). Something like `KUKA_DEG2RAD`?